include/ccmath/internal/builtins/fma.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <type_traits>
+
+namespace ccm::internal::builtin
+{
+	template <typename T>
+	inline constexpr bool has_constexpr_fma =
```

Comment:
Is there a way to make this look nicer and more readable? Maybe take the #if directive before the constexpr and just have two defs of the constexpr for readability?

Reply:
Sadly there is no real better way to make it look better, but it does eliminate the ugliness of all the macros we were using before.